bcdi/preprocessing/bcdi_utils.py

```diff
@@ -1391,6 +1393,16 @@ def show_rocking_curve(
         ),
     ].sum(axis=(1, 2))
 
+    # Create mask to remove duplicate in tilt_values
+    mask = [
+        r not in np.delete(tilt_values.copy(), j) for (j, r) in enumerate(tilt_values)
+    ]
+
+    # Apply mask
+    rocking_curve = rocking_curve[mask]
+    tilt_values = tilt_values[mask]
+    nb_frames = np.sum(mask)
+
```

Comment:
These modifications are not in the scope of the pull request, address it in an issue if needed.